libexec/scrape

```diff
@@ -8,6 +8,8 @@ while [ $# -gt 0 ]; do
       OVERWRITE=true ;;
     -n | --dry-run )
       DRY_RUN=true ;;
+    --rbx | --rubinius )
+      RUBIES+=('rubinius') ;;
```

Comment:
I think we can safely say that 3.2 is our baseline. I don't think I've ever tested any of these codez in an earlier version of bash, so we can't really claim support. But, we can't use bash 4, which is a shame because it has plenty array goodies that would be useful.